### ISSUE TITLE
datetime64[ns] - unknown/unsupported

### ISSUE THREAD

**Issue by rafalbigaj** [ibm-watson-data-lab/pixiedust#644]
As a `data scientist`, I want to `use date time filed` so that `my data is visualised as a time series`

### Expected behavior

My pyspark DF has schema:
```
root
 |-- ts: timestamp (nullable = true)
 |-- metric: string (nullable = true)
 |-- usage: double (nullable = true)
```
I'd like to use it to visualise time series.

### Actual behavior

I'm not able to use `ts` filed which is indicated as:
```
ts     unknown/unsupported
```

The data frame is interpreted by pixiedust as:
```
Field types:
ts: datetime64[ns]
metric: object
usage: datetime64[ns]
```

![screen shot 2018-03-14 at 08 39 17](https://user-images.githubusercontent.com/824894/37389103-3da4615c-2763-11e8-96c1-a10456e86044.png)

![screen shot 2018-03-14 at 08 40 04](https://user-images.githubusercontent.com/824894/37389122-55bbfeb2-2763-11e8-98aa-e8e6d5d6eb72.png)

### Steps to reproduce the behavior

The original data set (parquet):
https://ibm.box.com/shared/static/51kcmtifwjikzt2wkrch1s252os7tyy2.tgz

Steps to prepare DF:
```python
from pyspark.sql.functions import to_utc_timestamp, avg

monitoringData = spark.read.parquet("wml-perf-data/wml-monitoring-metics.2018-03-13_12.14.14.parquet")

metrics = monitoringData \
    .withColumn("ts", to_utc_timestamp(monitoringData.timestamp, "PST")) \
    .groupBy("ts", "metric").agg(avg(monitoringData.value).alias("usage"))
```

**Comment by DTAIEB** [comment]
Good catch @rafalbigaj. Fix has been provided and will be available in the next release of PixieDust, I verified that the datetime series is now properly visualized:

![image](https://user-images.githubusercontent.com/4912140/37410499-28ef0c62-2777-11e8-9861-6ec17134596e.png)

**Comment by rafalbigaj** [comment]
Thanks a lot @DTAIEB, when would it be available?

**Comment by DTAIEB** [comment]
@rafalbigaj Next release of PixieDust is currently scheduled for Friday March 30th.
You can already test the fix by installing directly from GitHub:
!pip install -U -e git+https://github.com/ibm-watson-data-lab/pixiedust#egg=pixiedust
When done, restart the kernel and refresh the page

**Comment by DTAIEB** [comment]
Fix has been released on PyPI with 1.1.9

**Comment by rafalbigaj** [comment]
Thanks!
